pennylane/ops/__init__.py

```diff
@@ -34,14 +34,59 @@
     by the plugin device.
 
 
+Architecture specific operations
+--------------------------------
+
 .. rst-class:: contents local topic
 
 .. toctree::
     :maxdepth: 2
 
     ops/qubit
     ops/cv
+
+
+General observables
+-------------------
+
+Observables that can be used on both qubit and CV devices.
 """
+#pylint: disable=too-few-public-methods,function-redefined
 
 from .cv import *
 from .qubit import *
+
+
+from .cv import __all__ as _cv__all__
+from .cv import ops as _cv__ops__
+from .cv import obs as _cv__obs__
+
+from .qubit import __all__ as _qubit__all__
+from .qubit import ops as _qubit__ops__
+from .qubit import obs as _qubit__obs__
+
+from pennylane.operation import Observable, CVObservable
+
+
+class Identity(CVObservable, Observable):
+    r"""pennylane.ops.Identity(wires)
+    Observable value of the identity observable :math:`\I`.
```

Comment:
sounds awkward ("observable" repeated twice)

Reply:
Oops. This must have been part of a find-all-replace-all, where the first word very obviously used to be `Expectation`.
```suggestion
    The identity observable :math:`\I`.
```